sdk/python/feast/infra/provider.py

```diff
@@ -144,7 +146,30 @@ def get_provider(config: RepoConfig, repo_path: Path) -> Provider:
 
         return LocalProvider(config, repo_path)
     else:
-        raise ValueError(config)
+        if "." not in config.provider:
```

Comment:
What if I provided the wrong name, like `azure`? should we have a `NotImplementedError` for not specifying a real "core" provider?

Reply:
We can reserve specific names and raise NotImplementedError for now. Otherwise it'll just complain that we can't parse the provider

Reply:
Ok